pkg/pmm/container.go

```diff
@@ -113,7 +87,7 @@ func pmmEnvs(cr *apiv1alpha1.PerconaServerMySQL, secret *corev1.Secret, dbType s
 		},
 		{
 			Name:  "PMM_AGENT_CONFIG_FILE",
-			Value: "/usr/local/percona/pmm2/config/pmm-agent.yaml",
+			Value: "/usr/local/percona/pmm/config/pmm-agent.yaml",
 		},
```

Comment:
Hi, since it's default value even without env variable, what's the reason to override it?

Reply:
this change will drop pmm2 support by the operator

Reply:
correct, we are not going to support PMM2 on the PS operator. We will keep support both for PMM2 and PMM3 for all the other operators i.e. pxc, psmdb and pgo

Reply:
cc @hors

Reply:
@BupycHuk PS operator will be GA only in the end of Q3, so we decided to release GA only with PMM3 support